examples/knative/python/kn-py-vm-attr/function.yaml

```diff
@@ -1,32 +1,26 @@
 ---
 apiVersion: v1
-kind: ConfigMap
+kind: Secret
 metadata:
-  name: vcconfig-cm
+  name: vcconfig-secret
+  labels:
+    app: kn-py-vm-attr
+type: Opaque
 data:
```

Comment:
data in secret must be base64 encoded. please see the other examples, e.g. powershell for kn, how to create secrets. since you're using multiple values, you might want to put this into a json structure so it can be used as a dict in your code.

Reply:
I did try to submit a change using `kustomize` to handle this configuration part. I hope it could help to clarify/simplify this step and provide an alternative way to handle configuration items.
Your PoV will be appreciated about this :)

Reply:
From the slack chat we add: I will provide a JSON configuration method.

Reply:
I did switch to JSON based config but I had to use `--env VCCONFIG_SECRET="$(cat kn-py-vm-attr_secret.json)"` in the Test step as docker expect only online variable values if you want to use `--env-file`.